visualization/app/codeCharta/codeCharta.api.model.ts

```diff
@@ -9,6 +9,7 @@ export interface ExportCCFile {
 	edges?: Edge[]
 	markedPackages?: MarkedPackage[]
 	blacklist?: ExportBlacklistItem[]
+	repoCreationDate?: string
```

Comment:
I should remove this property. It was added to be able to do some data analysis.
In the future we could ask the User for some more meta data like `repoCreationDate`, `application domain` and so on for a more detailed data analysis.

Reply:
repoCreationDate property is used in fileHelper thats why i do not think that we should remove it.